projects/1.18-fabric/assets/controlling/controlling/lang/zh_cn.json

```
@@ -5,10 +5,11 @@
   "options.showNone": "显示未绑定按键",
   "options.availableKeys": "可用按键",
   "options.sort": "排序",
-  "options.category": "分类",
+  "options.category": "类别",
   "options.key": "按键",
-  "options.sortNone": "空按键",
+  "options.sortNone": "无",

```

```suggestion
  "options.sortNone": "默认",
```
原文意思是「无」，但是放在排序中就是使用「默认」排序，不确定使用哪种来替代「空按键」